drivers/shared/executor/executor_universal_linux.go

```diff
@@ -75,7 +75,12 @@ func (e *UniversalExecutor) configureResourceContainer(pid int) error {
 		},
 	}
 
-	configureBasicCgroups(cfg)
+	err := configureBasicCgroups(cfg)
+	if err != nil {
+		e.logger.Debug("cgroup create", "error", err)
```

Comment:
This feels like a warning or an error - maybe something like.  Also would be nice to wrap the error.
```suggestion
		e.logger.Warn("failed to create cgroup", "error", err)
		return fmt.Errorf("failed to create cgroups: %v", err)
```